### DISCUSSION TITLE
Durées des missions "Gagner 5 points de rage..." et "Gagner 15 points de rage..."

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2706] [Nouvelles idées]
Hello,

Je n'ai pas trouvé de post similaire (désolé si la question a déjà été posée).
Serait-il possible d'inverser les durées des missions "Gagner 5 points de rage..." qui dure 10 jours et "Gagner 15 points de rage..." qui dure 7 jours ?
Ou alors...
Entre les deux missions, il y a "Gagner 10 points de rage..." qui dure elle aussi 10 jours. Peut-être passer la mission "15 points de rage" à 15 jours ?

Merci beaucoup!

**Comment by Ntalcme** [comment]
J'imagine bien la raison mais j'aimerais quand même demander pourquoi ?

**Reply by Ravenclaw-l** [reply]
Pourquoi quoi ?

**Reply by Ntalcme** [reply]
Vouloir changer.

**Reply by Ravenclaw-l** [reply]
Pour assurer une certaine logique dans la durée au vu de la difficulté des missions.

**Comment by BastLast** [comment]
Les récompenses sont calculées en fonction de la durée et difficulté donc moi je suis ok pr inverser mais ca veut dire moins de récompenses

**Reply by BastLast** [reply]
C'est fait expres que la plus dure soit plus courte hein c'est pas une erreur

**Reply by Ntalcme** [reply]
C'est exactement pour ça que je demandais pourquoi, c'est pas court pour rien.

**Reply by Ravenclaw-l** [reply]
D'acc.
Quand je regarde les "10 points", "25 points" et "100 points", elles suivent une progression logique avec une augmentation des récompenses. La "15 points" en revanche sort du lot. elle est nettement plus courte pour une différence de gain/difficulté limitée.

**Reply by BastLast** [reply]
> D'acc.
> Quand je regarde les "10 points", "25 points" et "100 points", elles suivent une progression logique avec une augmentation des récompenses. La "15 points" en revanche sort du lot. elle est nettement plus courte pour une différence de gain/difficulté limitée.
> 

Je checkerai à loccas go me reping ca en toute fin de v5

**Reply by Ravenclaw-l** [reply]
ça marche 👍🏼

**Reply by Procion06** [reply]
le gros point noir avec la "15 points" c'est qu'elle est difficilement réalisable comparée aux autres

à moins de bien connaître les choix pour choper la rage et d'avoir de la chance, il faut faire 3 runs, donc déjà avoir les gemmes pour, et quand je regarde les profils des joueurs c'est pas gagné 
ensuite si elle tombe à cheval entre 2 semaines, ça laisse un délai vraiment très court pour enchainer les runs

**Reply by Ntalcme** [reply]
Après bien connaitre les choix ça doit pas rentrer dans l'équation.
Sinon je comprends l'argument mais faut dire que la mission rapporte pas mal, on pourrait refixer le temps mais ça baisserait la récompense car deviendrait plus simple.
Après on va pas se mentir, de manière globale toutes les missions (secondaires/daily) sont à revoir et les suggestions sont les bienvenues comme toujours.

**Reply by Procion06** [reply]
si ça rentre forcément dans l'équation 
on fait régulièrement des runs à 4/5 ou 6 de rage en connaissant les choix donc imagine pour tous les joueurs qui ne les connaissent pas
faut bien prendre en compte que tout le monde n'a pas la même expérience du jeu ni les mêmes moyens en fonction de son ancienneté 
et même les vieux joueurs n'ont pas tous 40 gemmes à dépenser 

qu'elle rapporte pas mal comme tu dis ou pas grand chose, à partir du moment où tu ne la valides pas, tu n'auras rien au final 
pour le coup ça ça ne rentre pas dans l'équation ^^
car je parle bien de la faisabilité, pas de la récompense

**Reply by Ntalcme** [reply]
L'expérience de jeu doit rentrer en compte de mon point de vue, tout comme les choix des issues différents qu'on connait avec le temps, la manière de fight etc, on ne doit pas rendre une mission plus simple sous prétexte qu'elle est difficile si l'on connait mal le jeu.
Néanmoins, c'est vrai que si elle tombe mal et qu'on a peu de chances pendant les runs c'est complexe mais j'ai envie de dire que ce sont les aléas du jeu, tout n'est pas toujours bien comme on le voudrait, soit elle tombe mal soit il nous manque des gemmes soit nous n'avons pas de chances dans nos gains de points de rage, c'est le principe de l'aléatoire.

Pour la quantité de gemmes faut savoir que l'île n'a pas été design dans le but d'être faite 3 fois/semaine chaque semaine donc ça me parait normal, et encore une fois si on se retrouve avec peu de gemmes ça vient de notre propre gesion de dépense de celles-ci, on pourrait parler du gain de gemmes qui atteint des sommes ridicules certaines semaines mais c'est les aléas du jeu.

Donc si l'on regarde le souci vient soit de l'aléatoire soit d'une méconnaissance du jeu ça ne me semble pas si problématique que ça.

**Reply by Ravenclaw-l** [reply]
Tu dis : > ... mais faut dire que la mission rapporte pas mal, on pourrait refixer le temps mais ça baisserait la récompense car deviendrait plus simple. 

Sauf que, justement, ce que je veux signaler, c’est qu’elle est déjà pas aussi rentable en xp que les autres missions, même si elle est plus courte !! 🤦🏼‍♀️
Si tu compares, tu verras que pour chaque point de rage gagné, la mission offre moins d'xp que les autres. Il suffit de diviser l'xp total par les points de rage à gagner pour s'en rendre compte. 

**La mission "Gagner 15 points de rage" rapporte moins d'xp que "Gagner 10/25 points...."** Elle n'est rentable qu'en terme de points, sans que ce soit ultra significatif.

Si le but est de faire des missions plus difficiles et plus rentables, il faut revoir le gain d'xp et d'argents pour cette mission.

**Reply by Ntalcme** [reply]
Certes c'est elle qui offre le moins d'xp par point de rage mais si on regarde la 25 donne moins que la 10 ou la 5, et que dire de la 5 qui offre le plus gros ratio de points/rage derrière la 15.

Son ratio est de 72 qui est légérement inférieur aux autres mais ça se retrouve compensé par le meilleur ratio de gain de points par rapport aux autres, ça me semble pas mal.
Pour l'argent honnêtement j'y fais pas attention, j'imagine que oui là y a un truc à revoir.

La question c'est plutôt est-ce qu'on uniformalise le gain ou bien on laisse ce rapport de ratio de gain points/xp différents ?

Je préfère également qu'on garde une logique et donc qu'on fasse un système proportionelle mais son ratio d'xp est compensé par celui des points.
D'un côté elle est la plus basse, de l'autre la plus haute.

**Reply by Ravenclaw-l** [reply]
Je me suis penchée sur la mission 15 points de rage, qui, **à mes yeux**, ne semble pas rentable en termes d’xp et d’argent par rapport au temps qu’elle demande. Elle est difficile, pour toutes les raisons évoquées par Procion, et franchement, je ne la trouve pas du tout intéressante à ce niveau. Après, s’il y a d’autres missions qui posent problème, autant tout régler en même temps. C’est quand même plus agréable de pouvoir échanger efficacement sur ces points plutôt que de se voir répondre ["c’est pas court pour rien". ](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/2706#discussioncomment-11593373)
